addons/mass_mailing/static/src/scss/mass_mailing.scss

```diff
@@ -108,6 +108,54 @@
         .o_kanban_record {
             width: 350px;
         }
+        &.o_kanban_ungrouped {
+            padding: 0;
+            .o_mailing_list_kanban_grouped {
+                display: none!important;
```

Comment:
Already mentioned this on another of your PRs I think, shouldn't there be a space between the value and the "!important" ?